fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneCursorContinuation.java

```diff
@@ -83,14 +85,18 @@ public static LuceneCursorContinuation fromScoreDoc(ScoreDoc scoreDoc,
         if (partitionId != null) {
             builder.setPartitionId(partitionId);
         }
-        if (partitionTimestamp != null) {
-            builder.setPartitionTimestamp(partitionTimestamp);
+        if (partitionKey != null) {
+            builder.setPartitionKey(ByteString.copyFrom(partitionKey.pack()));
+        }
+        if (searchAfterPrimaryKey != null) {
+            builder.setSearchAfterPrimaryKey(ByteString.copyFrom(searchAfterPrimaryKey.pack()));
         }
```

Comment:
I think you want to get this from the `scoreDoc`. It should always be the second field in the ordering.

Reply:
right